HeterogeneousCore/SonicTriton/test/BuildFile.xml

```diff
@@ -1,5 +1,8 @@
-<test name="TestHeterogeneousCoreSonicTritonProducerCPU" command="${LOCALTOP}/src/HeterogeneousCore/SonicTriton/test/unittest.sh ${LOCALTOP} CPU"/>
-<test name="TestHeterogeneousCoreSonicTritonProducerGPU" command="${LOCALTOP}/src/HeterogeneousCore/SonicTriton/test/unittest.sh ${LOCALTOP} GPU"/>
+<test name="TestHeterogeneousCoreSonicTritonProducerCPU" command="unittest.sh ${LOCALTOP} CPU"/>
+<test name="TestHeterogeneousCoreSonicTritonProducerGPU" command="unittest.sh ${LOCALTOP} GPU"/>
+<test name="TestHeterogeneousCoreSonicTritonVersionCheck" command="cmsTritonConfigTool versioncheck">
+  <use name="cmsswdata"/>
+</test>
```

Comment:
@kpedro88 After this PR has been merged, if it wasn't already in your plans, could you make a test that tries to use an incorrect version/checksum in `cms-data` to make sure this check actually flags that?

Reply:
Yes, this is a good idea.